### DISCUSSION TITLE
Is it legal for pcqm4m-lsc to use atom coordinates computed by rdkit?

### DISCUSSION THREAD

**Discussion by zouxiaochuan** [snap-stanford/ogb#145] [PCQM4M-LSC]
i happend to find that rdkit can compute atom coordinates from smiles string, is that legal to use?

**Comment by weihua916** [comment]
Yes, you can use any functionality of the rdkit, as long as the test inference time is within 12 hours. See details [here](https://ogb.stanford.edu/kddcup2021/rules/#pcqm4m_time).

**Reply by joneswong** [reply]
hi weihua, I embed the molecules into 3D by rdkit and apply some DFT APIs, but the resultant gaps are far from what you have provided. Is it depend on the original 3D positions you used, Or the organizers just calculate the DFT by 2D positions (i.e., z=0 for all atoms)? BTW, did you add Hs to the molecules by rdkit before applying DFT, or Hs are removed? Thanks! I think it would be helpful for competitors to know more about how did you calculate the ground-truth gap.

**Reply by weihua916** [reply]
Hi! My guess is that it is because 3D structure calculated by rdkit is not accurate. Also, hyper-parameters of the DFT may be different. We use expensive DFT to obtain accurate 3D structures. More details can be found in [1]. 

[1] Nakata, M., & Shimazaki, T. (2017). PubChemQC project: a large-scale first-principles electronic structure database for data-driven chemistry. Journal of chemical information and modeling, 57(6), 1300-1308.
